src/main/webapp/css/srp_style.css

```diff
@@ -98,3 +102,21 @@ pre::-webkit-scrollbar {
   margin-right: 5px;
   vertical-align: bottom;
 }
+
+.vote {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 20px;
+  margin-right: 10px;
+}
+
+.vote button {
+  cursor: pointer;
+  background: none;
+  border: none;
+}
+
+.total-votes {
+  font-size: 15px;
```

Comment:
If you remove this, you can get rid of the span and so forth in the HTML, making it simpler. The font will then be 16px, making it consistent with the header.

Reply:
I still need to get access to total votes in the script.js file though so I don't think I can get rid of the span completely?